### PR TITLE
LL-8243 playwright reporting

### PR DESCRIPTION
**Pull request by valpinkman**
## 🦒 Context (issues, jira)

Adds reporting to PR about failing tests and screenshots etc....

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [x] a specific test planned is defined on Jira
  - [x] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [x] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

### CODE REVIEW

**Comment by valpinkman** [comment]
So the work in this PR is not atomic. I will need to have it merge to develop to continue testing and setting it up (yeah what a pain).

File: mylog.txt [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+[31m    account.spec.ts:13:5 › Accounts › [ETH] Add account [90m============================================[31m[39m
+[31m    account.spec.ts:13:5 › Accounts › [ATOM] Add account [90m===========================================[31m[39m
+[31m    account.spec.ts:13:5 › Accounts › [XTZ] Add account [90m============================================[31m[39m
+[31m    account.spec.ts:13:5 › Accounts › [XRP] Add account [90m============================================[31m[39m
+[31m    manager.spec.ts:8:1 › Manager [90m==================================================================[31m[39m
+[31m    onboarding.spec.ts:6:1 › Onboarding [90m============================================================[31m[39m
+[31m    passwordlock.spec.ts:12:1 › Enable password lock [90m===============================================[31m[39m
+[31m    settings.spec.ts:7:1 › Settings [90m================================================================[31m[39m
+error Command failed with exit code 1.
+info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
Comment (MortalKastor) [inline mylog.txt:311]: Should this file be committed?

**Review by MortalKastor — DISMISSED** [review]

File: mylog.txt (hunk 1 above)
Comment (valpinkman) [reply mylog.txt:311]: NOPE

**Pull request merged**